src/tests/banners/DefaultContributionsBannerTest.ts

```diff
@@ -0,0 +1,53 @@
+import fetch from 'node-fetch';
+import {
+    BannerTestGenerator,
+    BannerContent,
+    BannerTest,
+    BannerTargeting,
+    BannerPageTracking,
+} from '../../types/BannerTypes';
+
+export const DefaultContributionsBannerPath = 'contributions-banner.js';
+
+const defaultBannerContentUrl =
+    'https://interactive.guim.co.uk/docsdata/1CIHCoe87hyPHosXx1pYeVUoohvmIqh9cC_kNlV-CMHQ.json';
+
+const DefaultContributionsBanner = (bannerContent: BannerContent): BannerTest => {
+    return {
+        name: 'DefaultContributionsBanner',
+        bannerType: 'contributions',
+        testAudience: 'NonSupporters',
+        canRun: (targeting: BannerTargeting, pageTracking: BannerPageTracking): boolean =>
+            // Do not serve to frontend for now
```

Comment:
frontend has its own rendering for this banner, so no point trying to get this implementation working on that platform now